Tests/FeaturevisorSDKTests/InstanceTests.swift

```diff
@@ -4,6 +4,21 @@ import XCTest
 @testable import FeaturevisorTypes
 
 class FeaturevisorInstanceTests: XCTestCase {
+    
+    func testEncodeDoesNotThrow() throws {
```

Comment:
The name of this test it too generic. Based on this name we don't know what we encode. Remove "throws" too, as this is a thing which you want to assert